code/game/objects/effects/landmarks.dm

```diff
@@ -167,10 +167,6 @@ INITIALIZE_IMMEDIATE(/obj/effect/landmark)
 	name = "Research Director"
 	icon_state = "Research Director"
 
-/obj/effect/landmark/start/geneticist

```

Comment:
Remember to edit the station maps .dmms.

Reply:
dun, I did this pr at a late time and i left map changes for when i woke up